### PR TITLE
ln command

### LINKED ISSUE #184: ln
Claiming ln (links)

### PR DESCRIPTION
**Pull request by connorkrupp**
# Advanced HW 9 submission

Closes #184 

Name: Connor Krupp
Uniqname: kruppcon

### CODE REVIEW

File: _commands/basics/ln.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+`ln` is used to create a link (symbolic or hard) to a file on disk. Default creates hard links.
+
+~~~ bash
+$ ln file1.txt file2.txt
+$ ln -s file1.txt file2.txt
+~~~
+
+<!--more-->
+
+### What is Linking?
```
Comment (mterwill) [inline _commands/basics/ln.md]: I really like this section! Nicely done. Would love to see a __So what do I want?__ section that describes you more often than not want to use symbolic links. i.e. if you don't know what to use, pick symbolic.

**Comment by mterwill** [comment]
Also this is something I always screw up. If you can try to capture the essence of the top answer here that'd be cool.

http://unix.stackexchange.com/questions/125132/ln-s-with-a-path-relative-to-pwd

Otherwise, looks great and I'm happy to merge after those two minor changes.

**Comment by connorkrupp** [comment]
Added those two sections

**Comment by mterwill** [comment]
Great!

**Pull request merged**